sdk/include/vcruntime/vcruntime.h

```diff
@@ -0,0 +1,215 @@
+
+#pragma once
+
+#include <_mingw.h>
+#include <sal.h>
+#include <vadefs.h>
+
+#define _CRT_STRINGIZE_(x) #x
+#define _CRT_STRINGIZE(x) _CRT_STRINGIZE_(x)
+
+#ifndef _MSC_VER
+#define __pragma(x) _Pragma(_CRT_STRINGIZE(x))
+#endif
+
+#ifdef __cplusplus
+    #define _CRT_BEGIN_C_HEADER \
+        __pragma(pack(push, _CRT_PACKING_IDENTIFIER, _CRT_PACKING)) \
+        extern "C" {
+    #define _CRT_END_C_HEADER \
+        } \
+        __pragma(pack(pop, _CRT_PACKING_IDENTIFIER))
+#else
+    #define _CRT_BEGIN_C_HEADER \
+        __pragma(pack(push, _CRT_PACKING_IDENTIFIER, _CRT_PACKING))
+    #define _CRT_END_C_HEADER \
+        __pragma(pack(pop, _CRT_PACKING_IDENTIFIER))
+#endif
+
+_CRT_BEGIN_C_HEADER
+
+#ifndef _CRTIMP
+ #ifdef CRTDLL /* Defined for ntdll, crtdll, msvcrt, etc */
+  #define _CRTIMP
+ #elif defined(_DLL)
+  #define _CRTIMP __declspec(dllimport)
+ #else /* !CRTDLL && !_DLL */
+  #define _CRTIMP
+ #endif /* CRTDLL || _DLL */
+#endif /* !_CRTIMP */
+
+#ifndef _VCRTIMP
+ #ifndef _VCRT_DEFINED_CRTIMP
+  #define _VCRTIMP _CRTIMP
+ #elif defined(_VCRT_BUILD) && defined(CRTDLL) && !defined(_VCRT_SAT_1)
+  #define _VCRTIMP __declspec(dllexport)
+ #else
+  #define _VCRTIMP
+ #endif
+#endif
+
+#ifndef _CRT_STRINGIZE
```

Comment:
This is defined on line 8, and reason why it is a guard here?

Reply:
That was an oversight. Fixed it.